index.html

```diff
@@ -191,9 +192,57 @@ <h2>About</h2>
                 </div>
 
             </section>
+    
+<!-- contact section -->
+<section class="contact_section">
+    <div class="mobile_logo"></div>
+    
+      <h4 class="title-contact text_header_font">Get Started</h4>
 
-          
+      <p class="section-desc text_body_font">
+        Hello!I am a software developer! I can 
+        help you build a product, feature or website.
+        Take a look of my works. If you like what you see 
+        and have a project you need coded, don't hesitate 
+        and contact me.
+      </p>
+      <form action="put-link-here" method="post">

```

Comment:
- [ ] Kindly add the [form service](https://formspree.io/html/) to collect data from forms
-   please check the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md)to see how to integrate this feature, currently when I try to submit the form I see `This page isn’t workingIf the problem continues, contact the site owner.
HTTP ERROR 405
`